packages/internal-scripts/package.json

```diff
@@ -35,6 +37,12 @@
       "output": [
         "lib"
       ]
+    },
+    "test": {
+      "command": "(cd ../../ && node packages/internal-scripts/bin/add-browser-exports.js test)",
+      "dependencies": [
+        "build"
+      ]
```

Comment:
nit: seems a bit misleading for a script called "test" here to be running a particular script.

can this be moved to a script on the monorepo package.json maybe
```json
  "scripts": {
    "test:package-exports": "wireit",
  },
  "wireit": {
    "test:package-exports": {
      "command": "add-browser-exports test", // adding to .bin allows this command
      "dependencies": {
        "./packages/internal-scripts:build",
      },
    },
  },
```
and add this to the wireit dependencies of `"test:not-starter-kits"`?

Reply:
I like this!

Reply:
Done! Thank you for helping me get it working (and in package-lock).